flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/TestUtils.java

```diff
@@ -52,10 +54,11 @@
 	 * @throws RuntimeException if none or multiple jars could be found
 	 */
 	public static Path getResourceJar(final String jarNameRegex) {
-		String moduleDirProp = System.getProperty("moduleDir");
-		Preconditions.checkNotNull(moduleDirProp, "The moduleDir property was not set, You can set it when running maven via -DmoduleDir=<path>");
+		// if the property is not set then we are most likely running in the IDE, where the working directory is the
+		// module of the test that is currently running, which is exactly what we want
+		Path moduleDirectory = MODULE_DIRECTORY.get(Paths.get("").toAbsolutePath());
```

Comment:
When you're taking commits from the master branch and backport them to 1.11, then it is customary to retain the authorship.